client/my-sites/domains/domain-management/edit-contact-info-page/types.tsx

```diff
@@ -0,0 +1,8 @@
+import { ResponseDomain } from 'calypso/lib/domains/types';
+
+export type EditContactInfoPageProps = {
+	domains: ResponseDomain[];
+	selectedDomainName: string;
+	// TODO: We should define the selectedSite object
+	selectedSite: Record< string, unknown > | boolean;
```

Comment:
There's a type for it in `calypso/state/ui/selectors/site-data`. 
You can do it like here: 

https://github.com/Automattic/wp-calypso/blob/52825ec8cc08c637896d7f3fb73dbf6222b04c13/client/components/domains/connect-domain-step/types/index.tsx#L19

Reply:
Ah cool! Thanks for pointing that out 🙇‍♂️ I'll update the code